# 异常类

class InitException(Exception):
  pass